gravitee-apim-repository/gravitee-apim-repository-hazelcast/pom.xml

```diff
@@ -39,6 +39,11 @@
 			<scope>provided</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>io.gravitee.platform</groupId>
+			<artifactId>gravitee-platform-repository-api</artifactId>
+		</dependency>
+
```

Comment:
Globally speaking when used in a plugin, should this dependency be in scope provided as is already provided by the container (gateway or management) ?